repo_id,repo_full_name,file,file_sha1,match
243260988,b1cat/CVE_2020_1938_ajp_poc,ajpy/__pycache__/ajp.cpython-37.pyc,66de61b89269868bfac567e81a2704209805001a,CVE-2020-1938
243260988,b1cat/CVE_2020_1938_ajp_poc,ajpy/__init__.pyc,6df84a2cf63d2fd68f81895398ee65a3ab31a0b2,CVE-2020-1938
243260988,b1cat/CVE_2020_1938_ajp_poc,README.md,a7ce6ab0b5b67523ed29eca9bc7feb8e1d75cf9e,CVE-2020-1938
243260988,b1cat/CVE_2020_1938_ajp_poc,ajpy/__pycache__/__init__.cpython-37.pyc,e10048a027922cd443c99c3016d22d7105cbe7aa,CVE-2020-1938
243260988,b1cat/CVE_2020_1938_ajp_poc,ajpy/ajp.pyc,f6ddbf2d964c3fa2905ca9c240bcfd979b0f42bd,CVE-2020-1938
243260988,b1cat/CVE_2020_1938_ajp_poc,_GITHUB_REPO_METADATA_,,CVE-2020-1938
